tool/src/webapp/templates/FolderPicker.html

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+
+<html rsf:id="html" xmlns="http://www.w3.org/1999/xhtml">
+	<head>
+		<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+		<META HTTP-EQUIV="CACHE-CONTROL" CONTENT="NO-CACHE"/>
+		<script rsf:id="scr=portal-matter" type="text/javascript" language="JavaScript" src="/library/js/headscripts.js"></script>
+		<script type="text/javascript" src="https://weblearn.ox.ac.uk/library/js/jquery/jquery-1.9.1.min.js"></script>
+		<script type="text/javascript" src="/library/editor/ckextraplugins/common/js/preload-ckeditor-assets.js"></script>
+		<script type="text/javascript" src="/library/editor/ckextraplugins/common-wl/js/file-tree.js"></script>
+		<script type="text/javascript" src="/library/editor/ckextraplugins/common-wl/js/folder-listing.js"></script>
+		<script type="text/javascript" src="/library/editor/ckextraplugins/common-wl/js/get-available-sites.js"></script>
+		<link rsf:id="scr=portal-matter" href="/library/skin/tool_base.css" type="text/css" rel="stylesheet" media="all" />
+		<link rsf:id="scr=portal-matter" href="/library/skin/default/tool.css" type="text/css" rel="stylesheet" media="all" />
+		<link rel="stylesheet" rsf:id="scr=rewrite-url" href="../css/Simplepagetool.css" type="text/css"/>
+		<link rel="stylesheet" href="../../library/editor/ckextraplugins/common-wl/css/file-tree.css" type="text/css"/>
+		<script type="text/javascript" language="JavaScript">
+			var relativePath;
+			$(function() {
+				var editFolderPath = $( "input[name='edit-folder-path']" ).val();
+				relativePath = $('#folder-path').val();
+				$('.listing').attr('data-directory', relativePath).folderListing({
+					onFolderEvent: function(folderCollectionId) {
+						var folder = folderCollectionId.attr('rel');
+						var directory = getDirectoryFromPath(folder);
+						updatePathInput(directory);
+				  	},
+				  	onFileEvent: function(file) {
+						window.open(file);
+				 	},
+				});
+				if( editFolderPath !== undefined){
+					updatePathInput(editFolderPath);
+				}	
+			});
+			function updatePathInput(data){
+				$('#active-folder').val(data);
+				$('#folder-path').val(data);
+				
+			}
+		</script>
```

Comment:
Can we put this in the js somewhere so that's the js is all together (for debugging, maintainence and possible testing)?